plugins/woocommerce/includes/admin/class-wc-admin-menus.php

```diff
@@ -246,29 +245,66 @@ public function menu_highlight() {
 				}
 				break;
 		}
+
+		// Highlight "Add new order" submenu when on the new order page.
+		$page   = isset( $_GET['page'] ) ? sanitize_text_field( wp_unslash( $_GET['page'] ) ) : ''; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		$action = isset( $_GET['action'] ) ? sanitize_text_field( wp_unslash( $_GET['action'] ) ) : ''; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+
+		if ( 'wc-orders' === $page && 'new' === $action ) {
+			$submenu_file = \Automattic\WooCommerce\Utilities\OrderUtil::get_order_admin_new_url(); // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
```

Comment:
Is there any risk of extensions trying to access `$submenu_file` too early and seeing a different path?